content/Assets/Styles/index.scss

```diff
@@ -24,4 +24,6 @@
 @import "components/all";
 
 @import "trumps/all";
+
+@import "../../node_modules/lightgallery.js/src/sass/lightgallery";
```

Comment:
Not a big deal but typically I'd say if we're importing third party styles they should be imported early (e.g before our components), allowing us to potentially have lightgallery-override components which modify these third party styles.